vmngclient/api/task_status_api.py

```diff
@@ -102,6 +32,21 @@ def __init__(self, session: vManageSession, task_id: str):
         self.url = f"/dataservice/device/action/status/{self.task_id}"
         self.task_data: List[SubTaskData]
 
+    def __validate_task(self):
+        WAIT_SECONDS = 10
+        REPEATS_NUMBER = 2
+        for _ in range(REPEATS_NUMBER):
+            task_data = ConfigurationDashboardStatusPrimitives(self.session).find_status(self.task_id)
+            if task_data.validation.status in (OperationStatus.VALIDATION_SUCCESS, OperationStatus.SUCCESS):
```

Comment:
can we scale `OperationStatus.VALIDATION_SUCCESS` and  `OperationStatus.SUCCESS` into one `Enum` object?

Reply:
Outdated